### ISSUE TITLE
Is there has val and avg attribute for data_time, top1 etc. in print format in main?

### ISSUE THREAD

**Issue by wakeelephant** [jiecaoyu/XNOR-Net-PyTorch#80]
if i % args.print_freq == 0:
            print('Epoch: [{0}][{1}/{2}]\t'
                  'Time {batch_time.val:.3f} ({batch_time.avg:.3f})\t'
                  'Data {data_time.val:.3f} ({data_time.avg:.3f})\t'
                  'Loss {loss.val:.4f} ({loss.avg:.4f})\t'
                  'Prec@1 {top1.val:.3f} ({top1.avg:.3f})\t'
                  'Prec@5 {top5.val:.3f} ({top5.avg:.3f})'.format(
                   epoch, i, len(train_loader), batch_time=batch_time,
                   data_time=data_time, loss=losses, top1=top1, top5=top5))

in this place, for top1, in my understanding, dor top1=top1 left top1 assign value to top1.val and right top1 assign value to top1.avg, but can this done?   does val and avg attrbute exit ?   thanks.  @jiecaoyu

**Comment by jiecaoyu** [comment]
Hi @wakeelephant , those parameters are updated [here](https://github.com/jiecaoyu/XNOR-Net-PyTorch/blob/ebccb503b2f9ebc96187a75b6daa63d95b61d110/ImageNet/networks/main.py#L346).
